src/database/connect.ts

```diff
@@ -1,6 +1,8 @@
 import { Sequelize } from 'sequelize-typescript';
 import { appConfig } from '../config';
 import logger from '../logger/pino.logger';
+import { UserEntity } from './entities/user.entity';
+import { TaskEntity } from './entities/task.entity';
```

Comment:
```suggestion
import { TaskEntity, UserEntity } from './entities';
```
Напрашивается `index.ts`